### ISSUE TITLE
Multilanguage interface (Italian)

### ISSUE THREAD

**Issue by bovirus** [erickutcher/httpdownloader#90]
@erickutcher 

How can I help you translating the GUI in Italian?
What file you need?

**Comment by uDEV2019** [comment]
1. download string_list.txt from /HTTP_Downloader/locale/
2. download locale_gernerator.exe from /HTTP_Downloader/locale_generator/Release/
3. translate the strings from txt file
4. run locale_generator.exe in the same directory where your translated string_list.txt is located
5. a file named it-IT must be appear and push this to repo

**Comment by bovirus** [comment]
Thanks for the info.

I believe that these info could be useful for translator and could be add to READ.me in the home of the repository.

**Comment by bovirus** [comment]
These the files in a zip archive

[httpdownloader-it.zip](https://github.com/erickutcher/httpdownloader/files/3575610/httpdownloader-it.zip)

that contains these files

- string_list.txt (with Italian translation)
- it-IT

**Comment by erickutcher** [comment]
Thanks for the contribution. I've added your Italian translation file to the repo and updated the Languages file. You can download that here: https://github.com/erickutcher/httpdownloader/releases/download/v1.0.2.7/Languages.zip

**Comment by bovirus** [comment]
@erickutcher 

Could be update the zip archive file here

https://erickutcher.github.io/

for 32bit/64bit platform

32bit - https://github.com/erickutcher/httpdownloader/releases/download/v1.0.2.7/HTTP_Downloader_32.zip
64bit - https://github.com/erickutcher/httpdownloader/releases/download/v1.0.2.7/HTTP_Downloader_64.zip

adding locale folder with three languages?

The the user that download it can get it directly ion the program.

**Comment by bovirus** [comment]
I found some issues on translation and GUI. 
I will post a report about it.

**Comment by bovirus** [comment]
@erickutcher

New language file
[httpdownloader-it.zip]

Some feedback/request for improvements about GUI

(https://github.com/erickutcher/httpdownloader/files/3578134/httpdownloader-it.zip)
![HTTP_Downlaoder_1 0 2 7_GUI_01a](https://user-images.githubusercontent.com/1262554/64323177-459d7f00-cfc4-11e9-83a5-a3434531ceb6.jpg)
![HTTP_Downloader_1 0 2 7_GUI_01b](https://user-images.githubusercontent.com/1262554/64323178-459d7f00-cfc4-11e9-8a22-fd7c2c8dc5e7.jpg)
![HTTP_Downloader_1 0 2 7_GUI_02a](https://user-images.githubusercontent.com/1262554/64323179-459d7f00-cfc4-11e9-8845-e8958dc9dc79.jpg)
![HTTP_Downloader_1 0 2 7_GUI_02b](https://user-images.githubusercontent.com/1262554/64323181-459d7f00-cfc4-11e9-8d9e-e9ffad5255f4.jpg)
![HTTP_Downloader_1 0 2 7_GUI_03](https://user-images.githubusercontent.com/1262554/64323183-459d7f00-cfc4-11e9-9585-c09dd595e521.jpg)
![HTTP_Downloader_1 0 2 7_GUI_04](https://user-images.githubusercontent.com/1262554/64323184-46361580-cfc4-11e9-9220-03c672316ad9.jpg)
![HTTP_Downloader_1 0 2 7_GUI_05a](https://user-images.githubusercontent.com/1262554/64323186-46361580-cfc4-11e9-948c-bda12c49a3de.jpg)
![HTTP_Downloader_1 0 2 7_GUI_05b](https://user-images.githubusercontent.com/1262554/64323189-46361580-cfc4-11e9-8b47-c482a26d15f7.jpg)
![HTTP_Downloader_1 0 2 7_GUI_06](https://user-images.githubusercontent.com/1262554/64323190-46ceac00-cfc4-11e9-8912-35b6db66e90e.jpg)

for ..a/b.jpg file where available b is the screenshot of new layout after changes.

**Comment by erickutcher** [comment]
Yeah I need to overhaul the layout a bit. I manually code in the GUI objects positions. Takes forever to get things right.

I keep the language files separate for personal preference. Whenever I get a program that has them or offers to install multiple languages, I usually just trash them. I'll probably just post a link to them on the main site.

**Comment by bovirus** [comment]
No problem about language,..

It's quite easy change string text file, compile as langauge file and test in place.

I will posta  feedback about default column order changes, and some proposal about string to make it more specific.
